resources/botania/lang/zh_cn.lang

```diff
@@ -0,0 +1,52 @@
+// 生成花朵
+
+tile.botania:flower.campanimia.name=坎帕尼米亚
+tile.botania:flower.campanimia.reference=真正的鉴赏家！
+botania.entry.campanimia=坎帕尼米亚
+botania.tagline.botania.entry.campanimia=来自精华的魔力
+botania.page.campanimia0=§1坎帕尼米亚§r 是一种喜欢吸收 thaumaturge 提炼器中剩余精华的花。留下的精华种类越多，它生成的 §4魔力§r 就越多。<br>请注意，提炼器必须正常工作——需要同时提供热量和水。
+botania.page.campanimia1=&o别喝得太醉哦&r
+
+tile.botania:flower.nuclianthus.name=纽克莲花
```

Comment:
```suggestion
tile.botania:flower.nuclianthus.name=核辐莲
```
All `zh_cn` localization for flower names seem be using transliteration and barely contains useful information. I guess this is bacause these names are artificial? Please provide what these names are made up of, or it might confuse translators a lot.

Reply:
Here are the names of flowers name reference:
Amuileria kaerunea (Aquilegia caerulea + Kaminari[thunder])
Antirrhift (Antirrhinum + Shift)
Echinacenko (Echinacea + kenko[health])
Jikanacea (Echinacea + jikan[time]) 
Rokku Eryngium (Eryngium + Rokku)
Campanimia (Campanula + alkimia)
Nuclianthus (Helianthus + nuclear)